### PR TITLE
unique: Fix #1047 and add test case accordingly.

### LINKED ISSUE #1047: octave symbolic unique
syms a;

unique([a,a])  works fine, but

unique([a])

leads to the following error:
error: Python exception: UnboundLocalError: local variable ‘s’ referenced before assignment
occurred at line 1 of the Python code block:
return sp.Matrix([list(uniq(*ins))]),
error: called from
pycall_sympy_ at line 178 column 7
unique at line 55 column 5

### PR DESCRIPTION
**Pull request by alexvong1995**
This should fix #1047. We also add a test case accordingly.

### CODE REVIEW

File: inst/@sym/unique.m [hunk 1]
```diff
@@ -67,3 +72,8 @@
 %! A = [1 2 3 3 4 5 5 6 7 7 x x y y];
 %! B = [1  2  3  4  5  6  7  x  y];
 %! assert (isequal (unique(A), B))
+
+%!test
+%! syms x
+%! A = [x]
```
Comment (cbm755) [inline inst/@sym/unique.m:78]: Maybe this is just `A = x`?

File: inst/@sym/unique.m [hunk 2]
```diff
@@ -47,12 +47,17 @@
 
 
 function r = unique(A)
+  cmd = {'(A,) = _ins'
+         'if A.is_Matrix:'
+         '    return sp.Matrix([list(uniq(*_ins))]),'
+         'else:'
+         '    return _ins'};
```
Comment (cbm755) [inline inst/@sym/unique.m:54]: I think we normally put this `cmd` after the `nargin` input processing (but could be wrong!)

**Review by cbm755 — COMMENTED** [review]
Thanks for the fix!  Please add your name and the year to the copyright header, decide if the above comments should be addressed, and then we'll get this merged!

File: inst/@sym/unique.m (hunk 2 above)
Comment (alexvong1995) [reply inst/@sym/unique.m:54]: I've checked other files in the same directory. This is indeed the convention.

File: inst/@sym/unique.m (hunk 1 above)
Comment (alexvong1995) [reply inst/@sym/unique.m:78]: Yep and thus it is actually not needed. We can just use `x`...

**Pull request merged**